dwh/validate_indicators.sh

```diff
@@ -64,5 +65,22 @@ if [[ "${counts}" != "34,13,0,4" ]]; then
     "expected to be '34,13,0,4' GOT ${counts}"
   exit 1
 fi
+
+# TX_NEW Test
+counts=$(cat ${TEMP_OUT} | awk -F, '
+    BEGIN {num_true = 0; num_false = 0; num_none = 0;}
+    /False,ALL-AGES_ALL-GENDERS/ {num_false=$6}
+    /True,ALL-AGES_ALL-GENDERS/ {num_true=$6}
+    /None,ALL-AGES_ALL-GENDERS/ {num_none=$6}
+    /True,25-49_male/ {num_male_25=$6}
+    END {printf("%d,%d,%d,%d", num_true, num_false, num_none, num_male_25); }')
+
+echo "Number of TX_NEW vs non-TX_NEW vs none: ${counts}"
+# TODO investigate 5->4 change before submit!
```

Comment:
And this was a note to myself to address before submitting [this PR](https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/pull/178) which I forgot; no need to repeat it :-)